docs/guides/install-c-api.mdx

```diff
@@ -49,7 +49,7 @@ This section provides build instructions for UNIX-like systems.
 
 Compilation requires the following tools:
 * A Rust compiler: see for example the [guide on installing Qiskit from source](/docs/guides/install-qiskit-source)
-* A C compiler: for example, GCC on Linux and Clang on MacOS
+* A C compiler: for example, GCC on Linux and Clang on MacOS. Qiskit's C API is compatible with a `C11 <https://en.wikipedia.org/wiki/C11_(C_standard_revision)>`__ compatible compiler.
```

Comment:
I wasn't sure of the best link for this. I decided on wikipedia but I don't think it's the best choice. The official ISO spec is not useful: https://www.iso.org/standard/57853.html and also costs a lot of money. There is: https://www.open-std.org/jtc1/sc22/wg14/www/standards.html which isn't super useful either but it does link to the standard: https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf Maybe it doesn't need a link at all. I'm fine to defer to whatever others think is best.

Reply:
Oops, I've been writing too many sphinx docs lately forgot to context switch to markdown.